pkgs/by-name/pl/plex-desktop/package.nix

```diff
@@ -121,7 +121,7 @@ let
   };
 in
 buildFHSEnv {
-  name = "${pname}-${version}";
+  name = "${pname}";
```

Comment:
```suggestion
  inherit pname version;
```
you want to do this

Reply:
Using `inherit meta` now, but I've kept the `name = pname` for the moment while I investigate an evaluation error.

Getting the error with "You must provide name if pname or version (preferred) is missing." when using `inherit meta pname;` at the very end (and no `name = pname;` at the top)

Reply:
Ah, it wants a version too. `inherit pname version meta;`fixed it.